### PR TITLE
Visual Testing Handbook translation to Spanish

### PR DESCRIPTION
**Pull request by delfilb**

### CODE REVIEW

**Comment by delfilb** [comment]
Hi @jonniebigodes I just finished translating the Visual Testing Handbook into Spanish.

I would like to know if I am following the correct process.
Thanks!

**Comment by jonniebigodes** [comment]
@delfilb this is so awesome. I'm going to take a look at it next week and if anything stands out, we'll gladly merge it. Sounds good? 

Stay safe

**Review by amartinez62 — APPROVED** [review]
LGTM 🎉

File: content/visual-testing-handbook/react/es/vtdd.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+Configuremos el proyecto de ejemplo usando [degit](https://github.com/Rich-Harris/degit) para descargar las plantillas repetitivas necesarias (aplicaciones parcialmente construidas con alguna configuración predeterminada). Ejecute los siguientes comandos:
+
+```shell
+# Clone la plantilla para este tutorial
+npx degit chromaui/visual-testing-handbook-react-template commentlist
+
+cd commentlist
+
+# Instale dependencias
+yarn
```
Comment (jonniebigodes) [inline content/visual-testing-handbook/react/es/vtdd.md:33]: @delfilb if you don't mind leaving the comments in the original form we'd appreciate it. We're aware that it brings the translation closer to home, but until we come up with a good solution for this we'd appreciate if you left them in English.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@delfilb a couple of items to take care of and this will be good to go. One of the items I left a comment and if you could adjust to the rest of the translation we'd appreciate it.

The other one is just a small change to prevent the translation from being reported as outdated. If you could adjust the following [item](https://github.com/chromaui/learnstorybook.com/blob/master/gatsby-config.js#L67) to the following in your fork:
```js
'visual-testing-handbook': {
   react: {
     en: 6.2,
     es: 6.2, // <-- add this
   },
},
```

And commit it we'd appreciate it. 

Let me know once the changes are done and we'll gladly merge this.

It goes without saying, thank you @amartinez62 for helping review the pull request, really appreciate it 🙏🏼.

Stay safe

**Comment by delfilb** [comment]
Hi @jonniebigodes!

Thanks for your feedback. I just finished with the requested changes. 
Please let me know if something else needs to be fixed. I will gladly do it.

Thanks again.
Stay safe!

**Comment by jonniebigodes** [comment]
@delfilb all good. Thanks for taking care of the changes, I'm going to merge and release this. Once again we really appreciate the time and effort you've put into this. Really stoked to get this version released and help out the community. 

One final thing, if you could pop into our [Discord Server]() and message me (same username) as I want to follow up with on one item. Sounds good?

Stay safe

**Pull request merged**